spring-cloud-gcp-examples/spring-cloud-gcp-integration-pubsub-example/README.adoc

```diff
@@ -8,13 +8,27 @@ The other app subscribes to that topic and logs received messages.
 == Setup & Configuration
 
 1. Configure your GCP project ID and credentials by following
-link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core[these instructions], or just let
-Spring discover those parameters for you.
-2. Go to the https://console.cloud.google.com/cloudpubsub/topicList[Google Cloud Console] and create
-a topic called `exampleTopic` and a subscription for that topic called `exampleSubscription`.
-3. Start the `spring-cloud-gcp-spring-integration-pubsub-example-sender` and
-`spring-cloud-gcp-spring-integration-pubsub-example-receiver` applications.
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-core[these instructions].
```

Comment:
Why don't use Internal links support in ASCII Doc?

Reply:
If you're referring to `<<>>`, it won't work because we're linking to outside this document.
It does work in the reference document because everything is compiled to the same final document.

Reply:
OK. I give up. I don't understand you, sorry.
What is the difference in your terms between `document` and ` reference document`?

In the same Spring Kafka we have several files for the documentation and when we build the final HTML or PDF those *internal links* are compiled properly to the actual links based on the actual version of the project.

Reply:
This is not included in the reference document, this is the README of the SI example, to be read from GitHub mostly.

Reply:
Ah! Got it! 
Sorry for my stupidity 😄 

I need to do something with my mind. It lets me down last time...

So, then your solution is OK.

LGTM